yarn-project/acir-simulator/src/avm/avm_context.ts

```diff
@@ -0,0 +1,79 @@
+import { Fr } from '@aztec/foundation/fields';
+
+/**
+ * Store's data for an Avm execution frame
+ */
+export class AvmContext {
+  /** - */
+  public readonly calldata: Fr[];
+  private returnData: Fr[];
+
+  // TODO: implement tagged memory
+  /** - */
```

Comment:
Note: in instances where what the variable does is obvious ive just filled these in